cmake/ExternalIntelHEXL.cmake

```diff
@@ -4,8 +4,8 @@
 FetchContent_Declare(
     hexl
     PREFIX hexl
-    GIT_REPOSITORY https://github.com/intel/hexl.git
-    GIT_TAG v1.0.1
+    GIT_REPOSITORY https://github.com/intel/hexl
+    GIT_TAG 1.1.0
```

Comment:
Could you please replace this tag name with a commit hash? Using a tag name forces CMake to always connect to GitHub even when the content is already downloaded and blocks CMake configuration without internet. Using commit hash does not require internet.

Reply:
Neat, I didn't know about this. Done.